elixir/apps/api/lib/api/controllers/actor_group_membership_controller.ex

```diff
@@ -0,0 +1,55 @@
+defmodule API.ActorGroupMembershipController do
+  use API, :controller
+  alias API.Pagination
+  alias Domain.Actors
+
+  action_fallback API.FallbackController
+
+  # List members for a given Actor Group
+  def index(conn, %{"actor_group_id" => actor_group_id} = params) do
+    list_opts =
+      Pagination.params_to_list_opts(params)
+      |> Keyword.put(:filter, group_id: actor_group_id)
+
+    with {:ok, actors, metadata} <- Actors.list_actors(conn.assigns.subject, list_opts) do
+      render(conn, :index, actors: actors, metadata: metadata)
+    end
+  end
+
+  # Update Actor Group Memberships
+  def update(
+        conn,
+        %{"actor_group_id" => actor_group_id, "memberships" => params}
+      ) do
+    add = Map.get(params, "add", [])
```

Comment:
Would it be better if we just accept a list of `%{actor_id: ...}` maps like changeset expects it? I think this will be more helpful with terraform provider that won't need to compare results and generate diffs and instead just apply to the state it expects. This is also a bit easier for syncs because third-party system won't need to compute the diff, but will be a bit harder when third-party system wants to do one-off update.

Reply:
Yeah, I see what you're saying.  I was thinking more in the one-off mindset when writing this.

What if we allowed both the current way and the way you're suggesting?

Reply:
That sounds good, maybe we can use different HTTP verb/path for that so that you can't mix both in the same request? Eg. send whole list in `PUT` and add/remove for `PATCH`?

Reply:
Yeah, separating them by HTTP verb would match up great.